tech-far-hub/src/components/mdxcontent.tsx

```diff
@@ -3,9 +3,10 @@ import { MDXProvider } from "@mdx-js/react";
 import { Alert } from "./alert";
 import { Summary } from "./summary";
 import ExternalResources from "./external-resources";
+import { ProcessList } from "./process-list";
 
 const MDXContent = ({ children }: { children: React.ReactNode }): JSX.Element => {
-  const components = { Alert, Summary, ExternalResources };
+  const components = { Alert, Summary, ExternalResources, ProcessList };
```

Comment:
This is so rad! I wondered how Docusaurus did this and didn't know it was possible in Gatsby!